src/components/Item.js

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+import { updateDoc, serverTimestamp, doc } from 'firebase/firestore';
+import { db } from '../lib/firebase';
+import './Item.css';
+
+function Item({ item, userToken }) {
+  const [checked, setChecked] = useState(false);
+  const [daysSincePurchased, setDaysSincePurchased] = useState();
+
+  useEffect(() => {
+    if (item.lastPurchased) {
+      const lastPurchasedSeconds = item.lastPurchased.seconds;
+      const dateNowSeconds = Date.now() / 1000;
+      const differenceInSeconds = dateNowSeconds - lastPurchasedSeconds;
+
+      setDaysSincePurchased(differenceInSeconds / 86400);
+    }
+  }, [item]);
+
+  useEffect(() => {
+    if (daysSincePurchased > 1) {
+      if (checked) {
```

Comment:
Nit: To reduce another logic check can we remove this if and always `setChecked(false)`? This should be a safe action since if `checked === true` it will become false and if `checked === false` it will stay false. Unless I'm missing something!